lab_1/main.py

```diff
@@ -69,9 +122,31 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance_list = []
+    left_context_list = []
+    right_context_list = []
+    operative_tokens = tokens[:]
+    word = word.lower()
+    while word in operative_tokens:
```

Comment:
This while loop will only be executed once, right? Because we remove `word` from `operative_tokens` at the 142 line. Or am I wrong?